### DISCUSSION TITLE
1/1

### DISCUSSION THREAD

**Discussion by giscus** [csfive/composing-programs-zh#35] [Announcements]
# 1/1

SICP Python 中文翻译

https://sicp.csfive.works/1/1

**Comment by mancuoj** [comment]
大家可以在评论区指出翻译的错漏之处 🐶

**Comment by lila233** [comment]
感谢，再也不用看机翻的感觉真好🆒

**Comment by gudo7208** [comment]
所有计算都始于三点：信息的表示、处理的逻辑、设计管理这个复杂逻辑的抽象，掌握这些基础知识需要我们去精确理解计算机程序的构造和解释。
这句话中改成：设计抽象来管理逻辑的复杂性，感觉会更好点？

**Reply by mancuoj** [reply]
确实，感谢，这就改 🐶
